sdk/core/core-client-rest/package.json

```diff
@@ -60,36 +59,73 @@
     "@azure/core-util": "^1.0.0",
     "@azure/abort-controller": "^2.0.0",
     "@azure/core-tracing": "^1.0.1",
-    "tslib": "^2.2.0"
+    "tslib": "^2.6.2"
   },
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@microsoft/api-extractor": "^7.31.1",
-    "@types/chai": "^4.1.6",
-    "@types/mocha": "^10.0.0",
+    "@microsoft/api-extractor": "^7.39.1",
     "@types/node": "^18.0.0",
-    "@types/sinon": "^17.0.0",
-    "chai": "^4.2.0",
-    "cross-env": "^7.0.2",
-    "eslint": "^8.0.0",
-    "inherits": "^2.0.3",
-    "karma-chrome-launcher": "^3.0.0",
-    "karma-coverage": "^2.0.0",
-    "karma-env-preprocessor": "^0.1.1",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-junit-reporter": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-mocha": "^2.0.1",
-    "karma-sourcemap-loader": "^0.3.8",
-    "karma": "^6.2.0",
-    "mocha": "^10.0.0",
-    "rimraf": "^3.0.0",
-    "sinon": "^17.0.0",
+    "@vitest/browser": "^1.2.1",
+    "@vitest/coverage-istanbul": "^1.2.1",
+    "eslint": "^8.55.0",
+    "playwright": "^1.41.0",
+    "prettier": "^3.2.4",
+    "rimraf": "^5.0.5",
+    "tshy": "^1.11.0",
     "typescript": "~5.2.0",
-    "ts-node": "^10.0.0"
+    "vitest": "^1.2.1"
   },
   "//metadata": {
     "migrationDate": "2023-03-08T18:36:03.000Z"
-  }
+  },
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
+      "browser",
+      "bun",
+      "deno",
+      "react-native"
+    ]
+  },
+  "imports": {
+    "#isReadableStream": "./src/helpers/isReadableStream.ts"
```

Comment:
I thought conditional imports are more for aliasing (or I think for controlling imports per runtime) - how come we needed them here?

Reply:
Removed import maps as not needed